docs/book/src/faq.md

```diff
@@ -26,3 +26,23 @@ It takes a few seconds for the federated identity credential to be propagated af
 ## What is the Azure Workload Identity release schedule?
 
 Currently, we release on a monthly basis, targeting the last week of the month.
+
+## What permissions are required to create a federated identity credential for Azure AD Application?
+
+Required roles:
```

Comment:
As in, you need all of these roles or a single one?

Reply:
Any of these would suffice. Updated it!